phys/module_sf_urban.F

```diff
@@ -1319,6 +1375,23 @@ SUBROUTINE urban(LSOLAR,                                           & ! L
        DRGDTB=DRGDTB1+DRGDTB2
        DRGDTG=DRGDTG1+DRGDTG2
 
+       IF (distributed_aerodynamics_option) THEN
```

Comment:
@epn09 It seems you replaced original TCP related quantity with TA, why is this? based on assumptions for your distributed parameter generation?

Reply:
@cenlinhe According to the materials that I read, in this scheme, canopy temperature is not calculated (hence, `TCP` and `TC` are unused). Heat from all surfaces (roof, wall, roads) are released to the upper atmosphere directly. Therefore, `TA` is used.

References:
* Section I of http://www.ide.titech.ac.jp/~kandalab/download/WRF_URBAN/add_improvements.html
* https://link.springer.com/article/10.1007/s10546-004-7956-x